tools/taskcluster/build_targets.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script configures what APKs are built using the options passed in
+from the taskcluster {{ event.version }}.
+
+{{ event.version }} should be in the form: <tag name>:[all,release,debug]-<platform name>:[arm,arm64,x86]
+Some examples of {{ event.version }} and the resultant output from this script
+
+This is the default behaviour with no options. Only the Release build of each
+architecture for each supported platform is built:
+$ python build_targets.py 1.1.4a
+assembleWavevrArmRelease assembleNoapiArmRelease assembleNoapiArm64Release assembleNoapiX86Release assembleSvrArmRelease assembleSvrArm64Release assembleOculusvrArmRelease assembleOculusvrArm64Release assembleGooglevrArmRelease assembleGooglevrArm64Release
+
+Specifies only build the Arm64 version of the OculusVR platform:
+$ python build_targets.py 1.1.4b-oculusvr:arm64
+assembleOculusvrArm64Release
+
+Specifies all build types including Release and Debug:
+$ python build_targets.py 1.1.4c:all
+assembleWavevrArm assembleNoapiArm assembleNoapiArm64 assembleNoapiX86 assembleSvrArm assembleSvrArm64 assembleOculusvrArm assembleOculusvrArm64 assembleGooglevrArm assembleGooglevrArm64
+
+Specifies Release build of Arm64 OculusVR, Arm WaveVR, and x86 NoAPI
+$ python build_targets.py 1.1.4d-oculusvr:arm64-wavevr:arm-noapi:x86
+assembleOculusvrArm64Release assembleWavevrArmRelease assembleNoapiX86Release
+
+Specifies Release and Debug builds of Arm64 OculusVR, Arm WaveVR, and x86 NoAPI:
+$ python build_targets.py 1.1.4e:all-oculusvr:arm64-wavevr:arm-noapi:x86
+assembleOculusvrArm64 assembleWavevrArm assembleNoapiX86
+"""
+import subprocess
+import sys
+
+platforms = {
+   'oculusvr': ['arm', 'arm64'],
+   'wavevr': ['arm'],
+   'googlevr': ['arm', 'arm64'],
+   'noapi': ['arm', 'arm64', 'x86'],
+   'svr': ['arm', 'arm64'],
+}
+
+def findMode(arg):
+   values = arg.split(':')
+   if len(values) > 1:
+     mode = values[1].lower()
+     if mode == 'r' or mode == 'release':
```

Comment:
I like the verbosity of this, but if you want to save a little code, could check `if mode[0] == 'r'` instead, etc.

Reply:
I actually thought about dropping the single letter option to force verbosity.

Reply:
heh, that's funny. probably fine to leave as is.